### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by ariel11** [maleck13/readline#26]
Hi, does license "BSD" mean the BSD-3-clause (https://opensource.org/licenses/BSD-3-Clause), or one of the other BSD licenses?  Thanks!

**Comment by bartvanandel** [comment]
Good point. We're currently building an app and are working on proper attribution. So far, the only (indirect) dependency that we have that doesn't have an SPDX-compatible license id is readline. It would really help if your package were using one of the identifiers from the [SPDX licenses](https://spdx.org/licenses/) list.

**Comment by fbeaudoincoveo** [comment]
I second that.

We have automatic validations to ensure that all of our project dependencies have a license that matches one of those allowed by our legal department. Unfortunately, "BSD" is considered too vague to be added to that list, which means that unless it is clarified, we'll have to either switch to another similar library that has a clear license, or implement the features we need on our own.

Would it be possible to use one of the SPDX identifiers instead?

https://spdx.org/licenses/

Thanks!

**Comment by lnqs** [comment]
Same for us. We're totally willing to live with not having a valid SPDX identifier, but knowing what BSD flavor this package is meant to be published as would be great.

**Comment by knowlsie** [comment]
Spoke with author who has relicensed it to Apache 2.0

**Comment by bartvanandel** [comment]
So, #32 fixed it then (nice!), however @maleck13 would need to release a new version of this package to make it active in NPM.
